packages/e2e-tests/tests/repaint.test.ts

```diff
@@ -18,8 +23,8 @@ test("repaint: repaints the screen screen when stepping over code that modifies
 
   const prevDataUrl = await getCanvasDataUrl(page);
 
-  await stepOver(page);
-  await waitForPaused(page);
+  await addBreakpoint(page, { lineNumber: 55, url: exampleKey });
+  await resumeToLine(page, 55);
```

Comment:
This seems suspicious for the same reasons as I mentioned in #10189 and #10187.


Chromium's stepping behavior is currently broken IMO (see RUN-2958, BAC-4405).

I don't think we should change test stepping behavior to match Chromium because (I think) that would be updating the tests to verify/approve broken behavior. This change does that.

Reply:
I'm okay changing the behavior here because this stepping issue is caught by the stepping tests and in general we should try to test one thing per test. Obviously this is impossible to do in an e2e test, but i think it's a reasonable goal.